src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,9 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {
+        return boolean ? 'yes' : 'no';
+      
+      console.log(booleanToWord(true)); 
+      console.log(booleanToWord(false));
+  };
+
```

Comment:
Few things here, you should always remove console logs when submitting especially debug type console logs like these. Also the formatting could be improved, you should either use 2 or 4 spaces throughout your projects and since you more seem to be using 2 than 4 Ill base suggestions throughout the review using that. Theres no hard and fast rule regarding that, just 2 seems to be more common in my experience
```
const booleanToWord = (boolean) => {
  return boolean ? 'yes' : 'no';
};
```